src/app/image-group-page/image-group-page.component.ts

```diff
@@ -365,96 +361,6 @@ export class ImageGroupPage implements OnInit, OnDestroy {
     return totalTime/intervals
   }
 
-  private getPPT(): void{
```

Comment:
This function was nearly identical to `getZip`. Combined into one function called `executeBulkExport` a little bit farther below